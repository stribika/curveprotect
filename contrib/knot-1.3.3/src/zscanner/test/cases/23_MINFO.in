$ORIGIN	.
$TTL	1

; For more tests on dname see NS test (same processing)

; OK
@	MINFO	.	.		; The simplest dnames
@	MINFO	@	@		; Use origin
@	MINFO	mail	mail.tld.	; Relative and absolute dnames
@	MINFO	\# 2 00 00		; Hexadecimal rdata
@	TYPE14	\# 2 00 00		; TYPE + Hexadecimal rdata
@	TYPE14	@	@		; TYPE
@	minfo	@	@		; Type in lower-case

; KO
@	MINFO
@	MINFO				; Empty rdata
@	MINFO	\# 0			; Hex empty rdata
@	MINFO	\#			; Missing hex length
@	MINFO	\# 3 00 0001		; Too long rdata
@	MINFO	\# 3 00 00		; Bad rdata length
