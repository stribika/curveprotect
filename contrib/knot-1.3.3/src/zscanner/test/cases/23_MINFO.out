OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=046D61696C00 046D61696C03746C6400
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000E
RDATA=00 00
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
